### ISSUE TITLE
NFT is work for DDPM, I have tested. But I have some question. please help me.

### ISSUE THREAD

**Issue by shoutOutYangJie** [NVlabs/DiffusionNFT#18]
I find the "r' variable is normalized to (0, 1) ,but most of "r"  values around 0.5. I print "r" variable.

> normalized_advantages_clip = (advantages_clip / config.train.adv_clip_max) / 2.0 + 0.5 
 r = torch.clamp(normalized_advantages_clip, 0, 1)

<img width="790" height="120" alt="Image" src="https://github.com/user-attachments/assets/b5b839ac-e336-4440-8891-c994c334f40f" />

most "r" ranges from (0.4, 0.6) not (0, 1) 
What was the reason for you to design it this way?

**Comment by ChenDRAG** [comment]
We mostly follow the pre-setup of FlowGRPO and do not test the reward shaping option much.  Could try to normalize reward to range (0,1) and see what happens
